scripts/mvn-install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -u -e -o pipefail
+
+# Find all the maven packages in the repo
+readonly DEPLOY_LABELS=$(bazel query --output=label 'kind("maven_publish rule", //...) - attr("tags", "\[.*do-not-publish.*\]", //...)')
+for pkg in $DEPLOY_LABELS ; do
+  bazel run "$pkg" --define=maven_repo="file://$HOME/.m2/repository"
+done
```

Comment:
Does this script help us use these packages locally?

Reply:
Precisely! Same as doing `gradle publishToMavenLocal` or `mvn install`

Reply:
nice!! we needed this. 

Is it possible to add some docs for this in either this PR or another